### PR TITLE
broken links

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 1]
```diff
@@ -43,7 +43,7 @@ https://runtime-fabric.s3.amazonaws.com ✔
 tcp://dias-ingestor-nginx.prod.cloudhub.io:443 ✔
 ```
 
-If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have added the required port IPs and hostnames to the allowlist as described in xref:install-port-reqs#port-ips-and-hostnames-to-add-to-the-allowlist[Port IPs and Hostnames to Add to the Allowlist].
+If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have added the required port IPs and hostnames to the allowlist as described in xref:install-port-reqs#port-ip-addresses-and-hostnames-to-add-to-the-allowlist[Port IP Addresses and Hostnames to Add to the Allowlist].
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/troubleshoot-guide.adoc]: Should `xref:install-port-reqs#port-ip-add...` be `xref:install-port-reqs.adoc#port-ip-add...` ?

Comment (wwarner-salesforce) [reply modules/ROOT/pages/troubleshoot-guide.adoc]: Also at https://github.com/mulesoft/docs-runtime-fabric/tree/v1.8/modules/ROOT/pages I do not see a file called install-port-reqs.adoc. Should it be install-prereqs.adoc?

Comment (IsaacEldridge) [inline modules/ROOT/pages/troubleshoot-guide.adoc]:
```suggestion
If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have added the required port IPs and hostnames to the allowlist as described in xref:install-prereqs#port-ip-addresses-and-hostnames-to-add-to-the-allowlist[Port IP Addresses and Hostnames to Add to the Allowlist].
```

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 2]
```diff
@@ -43,7 +43,7 @@ https://runtime-fabric.s3.amazonaws.com ✔
 tcp://dias-ingestor-nginx.prod.cloudhub.io:443 ✔
 ```
 
-If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have added the required port IPs and hostnames to the allowlist as described in xref:install-port-reqs#port-ips-and-hostnames-to-add-to-the-allowlist[Port IPs and Hostnames to Add to the Allowlist].
+If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have added the required port IPs and hostnames to the allowlist as described in xref:install-prereqs#port-ip-addresses-and-hostnames-to-add-to-the-allowlist[Port IP Addresses and Hostnames to Add to the Allowlist].
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/troubleshoot-guide.adoc]:
```suggestion
If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have added the required port IPs and hostnames to the allowlist as described in xref:install-prereqs.adoc#port-ip-addresses-and-hostnames-to-add-to-the-allowlist[Port IP Addresses and Hostnames to Add to the Allowlist].
```

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**